source-hubspot-native/source_hubspot_native/models.py

```diff
@@ -112,6 +119,22 @@ class Properties(BaseDocument, extra="forbid"):
     results: list[Property]
 
 
+class DealPipeline(BaseDocument, extra="allow"):
+
+    createdAt: AwareDatetime | None
+    updatedAt: AwareDatetime | None
```

Comment:
Will these fields only ever be `None` for tombstones?

Reply:
Yep, assuming the API doesn't do anything strange. But they should always be present in actual API objects.